### PR TITLE
[FIX] tests: remove memory limit for chrome headless

### PR DESCRIPTION
**Pull request by d-fence**
Since Chrome version 80, the V8 javascript engine is using an new
pointer compression feature [1]. This features tries to reserve more the
4GiB of memory. As a consequence, during HttpCase tests, odoo tries to
launch chrome headless in a subprocess but fails because of the
memory-limit-soft.

With this commit, the memory limits are removed just before launching
Chrome and restored right after.

Also:
  * a new Chrome CLI switch is used to prevent crash reports to be sent
  to google
  * a chrome CLI switch is used to disable gpu usage
  * a bug that only occurs during Chrome crash is fixed [2]

[1] https://v8.dev/blog/v8-release-80
[2] None type object does not have a send method

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by d-fence** [comment]
@odony a more simple solution than #45347

File: odoo/tests/common.py [hunk 1]
```diff
@@ -558,17 +558,25 @@ def _chrome_start(self):
...
             '--remote-debugging-port': str(self.devtools_port),
             '--no-sandbox': '',
+            '--disable-crash-reporter': '',
+            '--disable-gpu': '',
         }
         cmd = [self.executable]
         cmd += ['%s=%s' % (k, v) if v else k for k, v in switches.items()]
         url = 'about:blank'
         cmd.append(url)
+        if os.name == 'posix' and platform.system() != 'Darwin':
```
Comment (odony) [inline odoo/tests/common.py]: what about thread safety? we just don't care anymore?

**Comment by odony** [comment]
Oh, do we actually need to do this in 12.0? It's mostly developer convenience after all.

**Comment by d-fence** [comment]
Closing as it's not thread safe.

**Pull request closed without merging**